lib/lib-storage/src/types.ts

```diff
@@ -1,4 +1,5 @@
 import { PutObjectCommandInput, S3Client, Tag } from "@aws-sdk/client-s3";
+import { AbortController } from "@aws-sdk/abort-controller";
```

Comment:
add this as a devDep in package.json?

Reply:
It's actually already there:

https://github.com/aws/aws-sdk-js-v3/blob/1098e30464355ffcd1a02166f0d748e646953eb8/lib/lib-storage/package.json#L37